gensim/models/word2vec.py

```diff
@@ -376,16 +376,17 @@ def __init__(
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-            default is 0 (off), useful value is 1e-5.
+            default is 1e-3, useful value is 1e-5, 0 stands for off.
```

Comment:
Should match word2vec.c recommendation – 'useful range is 0 to 1e-5` – rather than imply that the default is somehow not as useful as some other value.